monix-reactive/shared/src/main/scala/monix/reactive/Observable.scala

```diff
@@ -1581,8 +1581,9 @@ abstract class Observable[+A] extends Serializable { self =>
     *
     * @see [[mapTask]] for serial execution
     */
-  final def mapParallelUnordered[B](parallelism: Int)(f: A => Task[B]): Observable[B] =
-    new MapParallelUnorderedObservable[A, B](self, parallelism, f)
+  final def mapParallelUnordered[B](parallelism: Int)(f: A => Task[B])
+                                   (implicit os: OverflowStrategy[B] = OverflowStrategy.Default): Observable[B] =
```

Comment:
I don't like this indenting style for params, there's something that annoys me greatly about things that dangle on the right, which is why this codebase doesn't do it. I know IntelliJ IDEA does this by default, but it can be configured to not do it.

Please use normal indentation for this line.